src/core/index_meta.rs

```diff
@@ -233,7 +233,10 @@ impl InnerSegmentMeta {
 pub struct IndexSettings {
     /// Sorts the documents by information
     /// provided in `IndexSortByField`
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub sort_by_field: Option<IndexSortByField>,
+    /// The `Compressor` used to compress the doc store.
+    pub compressor: Compressor,
```

Comment:
considering we serialize this thing, 

`s/compressor/docstore_compression/g` ?

Reply:
yes that's better